### DISCUSSION TITLE
highlight error with onedarkpro

### DISCUSSION THREAD

**Discussion by towry** [olimorris/onedarkpro.nvim#204] [Q&A]
<img width="1266" alt="截屏2023-07-02 12 02 31" src="https://github.com/olimorris/onedarkpro.nvim/assets/8279858/4b5c5c6d-2e84-4b81-bb9e-1215d925c081">

**Comment by olimorris** [comment]
Can you explain a little more how you got this? This image doesn't help and a bug report would he ideal.
